iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/NetworkXml.java

```diff
@@ -215,6 +223,27 @@ private static ExtensionXmlSerializer getExtensionXmlSerializer(ExportOptions op
         return extensionXmlSerializer;
     }
 
+    private static String getNamespaceUri(ExtensionXmlSerializer extensionXmlSerializer, ExportOptions options) {
+        return getNamespaceUri(extensionXmlSerializer, options,
+                options.getVersion() == null ? IidmXmlConstants.CURRENT_IIDM_XML_VERSION : IidmXmlVersion.of(options.getVersion(), "."));
+    }
```

Comment:
Rewrite this method in two lines to make it simpler:
```
String version = options.getVersion() == null ? IidmXmlConstants.CURRENT_IIDM_XML_VERSION : IidmXmlVersion.of(options.getVersion(), ".");
return getNamespaceUri(extensionXmlSerializer, options, version);
```

Reply:
Why is the test done on the version on the IidmXmlVersion here?